### PR TITLE
[FAB-17664] Add/Remove channel policies

### PR DESCRIPTION
**Pull request by DereckLuo**
Signed-off-by: Chongxin Luo <Chongxin.Luo@ibm.com>

#### Type of change

- New feature

#### Description
* Implemented add/remove channel policies.

#### Related issues
[FAB-17664](https://jira.hyperledger.org/browse/FAB-17664)

### CODE REVIEW

File: pkg/config/policies.go [hunk 1]
```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
 	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName], policyName, policies)
 }
 
+// AddChannelPolicy modifies an existing channel policy configuration.
+// When the policy exists it will overwrite the existing policy.
```
Comment (caod123) [inline pkg/config/policies.go:253]: Is this the ideal behavior for Add? Not saying it's wrong but we should consider whether we should allow the user to overwrite existing policies via the Add method instead of a Replace method or via a Delete followed by an Add

File: pkg/config/policies.go [hunk 2]
```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
...
+func (c *ConfigTx) AddChannelPolicy(modPolicy, policyName string, policy Policy) error {
+	return addPolicy(c.updated.ChannelGroup, modPolicy, policyName, policy)
+}
+
+// RemoveChannelPolicy removes an existing channel policy configuration.
```
Comment (caod123) [inline pkg/config/policies.go]: Mentioning configuration at the end is a bit wordy. Probably good enough to say `RemoveChannelPolicy removes an existing channel level policy.`

**Review by caod123 — COMMENTED** [review]
small comments, mostly some function comments and considerations

File: pkg/config/policies.go [hunk 3]
```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
 	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName], policyName, policies)
 }
 
+// AddChannelPolicy modifies an existing channel policy configuration.
```
[the remark this answers is not in the document]
Comment (DereckLuo) [reply pkg/config/policies.go]: done

File: pkg/config/policies.go (hunk 1 above)
Comment (DereckLuo) [reply pkg/config/policies.go:253]: make sense, thought about the same thing during the implementation. 
However, currently all add methods in policies.go share the same implementation, it's probably worth it to consider the ideal behavior, and apply it over the entire package.

File: pkg/config/policies.go (hunk 2 above)
Comment (DereckLuo) [reply pkg/config/policies.go]: done.

**Comment by DereckLuo** [comment]
rebased.

File: pkg/config/policies.go (hunk 1 above)
Comment (sykesm) [reply pkg/config/policies.go:253]: Add != Set; Set could imply Add but not the other way round, IMO.

**Review by sykesm — APPROVED** [review]
